### ISSUE TITLE
Warbo wants mobile ticket to generate a png ticket file with qr

### ISSUE THREAD

**Issue by petterroea** [InfectedLan/InfectedTickets#25]

**Comment by halvors** [comment]
Maybe take a look at this before the event? Users may have been starting to print tickets now.

**Comment by petterroea** [comment]
No, push for next release. I have a idea for how we will solve it though. 

It requires warbo to design a "template" file for the png ticket, and if i ask for it now, he will tell me to focus on the compo page.

**Comment by halvors** [comment]
@petterroea Close this?

**Comment by petterroea** [comment]
Nah
